dangerfile.js

```diff
@@ -0,0 +1,71 @@
+const { danger, message, fail, warn, schedule } = require("danger");
+const { default: jiraIssue } = require("danger-plugin-jira-issue");
+// const { default: noConsole } = require("danger-plugin-no-console"); this plugin has some issues
+
+const PR_REVIEWS_CHECKS = Object.freeze({
+    BIG_PR_THRESHOLD: 700,
+    REMOVED_LINE_THRESHOLD: 200,
+    TEAMS_INITIALS: ["FW", "TD", "FF", "FWH", "FFH", "TDH"],
+    WARNS: {
+        TITLE: `Title required Issue number`,
+        CHECK_LOCK_FILE: `:exclamation: Changes were made to package.json, but not to package-lock.json - <i>'Perhaps you need to run \`npm install\`?'`,
+        WIP: `PR is classed as Work in Progress`,
+        EXCEED_THRESHOLD: `:exclamation: Your PR has over 700 lines of code, please consider breaking it down so we can effectively review it. :thumbsup:`,
+        MERGE_CONFLICTS: `Please merge development branch`,
+        ONLY_DEV_BRANCH: `:exclamation: Please re-submit this PR to development branch, we may have already fixed your issue. 🙏`,
+        MISSING_DES: `:exclamation: Please add a description to your PR to make it easier to review`,
+        MERGE_COMMITS: `Please rebase to get rid of the merge commits in this PR 🙏`,
+        FILE_ISSUE: `Please file the issue for development branch`,
+        PATH_WARN: `Make sure to file it for development branch as well`,
+        REVIEWER_REQ: `:exclamation: Please make sure to assign reviewer in this PR`,
+        PACKAGE_JSON_ERR: `Changes were made to package.json, but not to package-lock.json - <i>'Perhaps you need to run \`npm install\`?'`
+    },
+    SUCCESS: {
+        LINES_REMOVED : `:tada: The PR removed ${danger.github.pr.deletions} lines. :clap:`,
+        NEW_FILES: `New Files in this PR: \n - `
+    }
+});
+
+//Accept PR only for development branch
+if(danger.github.pr.head.ref == "master") return fail(PR_REVIEWS_CHECKS.WARNS.ONLY_DEV_BRANCH);
```

Comment:
At times we need to file PR for master branch

Reply:
in that case, it will only check this condition and return

if you want i will remove return from here

Reply:
in that case it will show only warning and then checks for other conditions also

Reply:
Ok so 
1. will it not fail the PR creation? 
2. The action bot will not say "checks failed"?

We can add warn and remove return

Reply:
1. will it not fail the PR creation? NO
2. The action bot will not say "checks failed"? adding warn